framework/ChiMesh/SweepUtilities/SweepBoundary/sweep_boundaries.h

```diff
@@ -11,97 +12,122 @@ namespace chi_mesh::sweep_management
 
 enum class BoundaryType
 {
-  VACUUM                = 0,
-  INCIDENT_HOMOGENOUS   = 1,
-  REFLECTING            = 2,
-  INCIDENT_HETEROGENOUS = 3
+  INCIDENT_VACCUUM                  = 0, ///< Zero for all angles, space
+  INCIDENT_ISOTROPIC_HOMOGENOUS     = 1, ///< One value for all angles, homogenous in space
+  REFLECTING                        = 2, ///< Reflecting boundary condition about a normal
+  INCIDENT_ANISOTROPIC_HETEROGENOUS = 3  ///< Complex different for each angle and face node
```

Comment:
What i I have a space-dependent isotropic boundary condition or a space-independent anisotropic boundary condition?

Reply:
I don't know what you have. Basically the `INCIDENT_ANISOTROPIC_HETEROGENOUS` can be used to emulate both `INCIDENT_VACCUUM` and `INCIDENT_ISOTROPIC_HOMOGENOUS`.

Reply:
This comment was mainly to highlight that under this option set, you have to have angle-space independence or angle-space dependence, but one dependent and one independent. I understand that with `INCIDENT_ANISOTROPIC_HETEROGENEOUS` you can specify anything.

Reply:
Not sure I understand the point here?